src/pages/Payment/ForexApproval/Reports/DeliquentItems.jsx

```
@@ -49,79 +48,50 @@ export default function DeliquentItems(props) {
+                              )
+                              .reduce((acc, val) => acc + val, 0)
+                      ).toLocaleString()}
                 </span>
               </div>
               <div className="">
                 <ul class="list-group">
-                  {summerizedPi
-                    ? props?.data?.summerized_pi?.items?.map((item) => (
+                  {props?.data?.items?.some((row) => row?.total_cost)
+                    ? props?.data?.items?.map((item) => (
                         <li class="list-group-item d-flex justify-content-between align-items-center task-order-name">
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)